cloneall

```diff
@@ -133,6 +133,14 @@ if [ -f $config_file ]; then
 else
 	errormsg "Repo configuration '$config_file' is not existing"
 fi
+
+# Receive event from other repos and update repository config file
+if [ -n "$REPOSITORY"  ] && [ -n "$PULL_REQUEST_BRANCH" ]; then 
+
+	sed -i "s/$REPOSITORY=\(.*\)/${REPOSITORY}=$PULL_REQUEST_BRANCH/" "$CONFIG_FILE"
```

Comment:
Variable for location of repo configuration should be `$config_file` (all lower-case characters).

Reply:
Thank you for your info. I have updated it.